e2e/support/helpers/e2e-embedding-helpers.js

```diff
@@ -261,3 +261,16 @@ export function createPublicQuestionLink(questionId) {
 export function createPublicDashboardLink(dashboardId) {
   return cy.request("POST", `/api/dashboard/${dashboardId}/public_link`, {});
 }
+
+export const visitFullAppEmbeddingUrl = ({ url, qs, onBeforeLoad }) => {
```

Comment:
The only conflict was with this util, because it was only recently introduce to `master`, so it wasn't in 49 branch.